build.sbt

```diff
@@ -63,15 +62,77 @@ lazy val checkerLibrary = (projectMatrix in file("checker-library"))
 lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(name := "akka-serializability-checker-plugin")
   .settings(commonSettings)
-  .settings(libraryDependencies ++= {
-    virtualAxes.value
-      .collectFirst { case x: ScalaVersionAxis => x.value }
-      .map {
-        case "2.13" => scalaPluginDeps213
-        case "2.12" => scalaPluginDeps212
-      }
-      .getOrElse(Seq.empty)
-  })
-  .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
+  .settings(
+    libraryDependencies ++= {
+      virtualAxes.value
+        .collectFirst { case x: ScalaVersionAxis => x.value }
+        .map {
+          case "2.13" => scalaPluginDeps213
+          case "2.12" => scalaPluginDeps212
+        }
+        .getOrElse(Seq.empty)
+    },
+    libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test, betterFiles % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val LocalMavenResolverForSbtPlugins = {
+  // remove scala and sbt versions from the path, as it does not work with jitpack
+  val pattern = "[organisation]/[module]/[revision]/[module]-[revision](-[classifier]).[ext]"
+  val name = "local-maven-for-sbt-plugins"
+  val location = userHome / ".m2" / "repository"
+  Resolver.file(name, location)(Patterns().withArtifactPatterns(Vector(pattern)))
+}
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
+  .enablePlugins(SbtPlugin)
+  .settings(name := "sbt-dumpschema")
+  .settings(commonSettings)
+  .settings(
+    pluginCrossBuild / sbtVersion := "1.2.8",
+    libraryDependencies ++= Seq(sprayJson, betterFiles),
```

Comment:
Hmmm actually... for clarity, what was the problem with including Borer in the classpath that using Spray fixed? 🤔

Reply:
Borer doesn't have JSON pretty print and Spray does